README.md

```diff
@@ -153,6 +154,35 @@ ID log injection with this environment variable:
 $ SIGNALFX_LOGS_INJECTION=true
 ```
 
+### Inject span tags into logs
+
+If log injection is enabled, in addition to trace ID and span ID, span tags can
+be injected to logs. By default only `service.name` is injected. To select which
+tags to inject, add `logInjectionTags` to init config:
+
+```javascript
+const tracer = require('signalfx-tracing').init({
+  tags: {environment: 'production', region: 'eu'},
+  // Only 'region' will be added to logs besides trace_id, and span_id
+  logInjectionTags: ['region']
+})
+```
+
+After which your logger will receive:
+```json
+signalfx: {
+  trace_id: <trace_id>,
+  span_id: <span_id>,
+  region: 'eu'
+}
+```
+
+Or set the same injected tag via an environment var:
+
+```bash
+$ SIGNALFX_LOG_INJECTION_TAGS=region
+```
```

Comment:
We should clarify which takes precedence. Code or env var.

Reply:
Added a clarification